packages/website/ts/components/wallet/wallet.tsx

```diff
@@ -114,7 +119,17 @@ const styles: Styles = {
         paddingTop: 8,
         paddingBottom: 8,
     },
-    accessoryItemsContainer: { width: 150, right: 8 },
+    accessoryItemsContainer: {
+        width: 150,
+        right: 8,
+    },
+    bodyInnerDiv: {
+        padding: 0,
+        // TODO: make this completely responsive
```

Comment:
Is this for a future PR?

Reply:
yup.. right now it is not completely responsive in the y-direction but is good enough for now